R/provider-claude.R

```diff
@@ -143,7 +143,7 @@ method(chat_body, ProviderAnthropic) <- function(
 
   if (!is.null(type)) {
     tool_def <- ToolDef(
-      fun = function(...) {},
+      function(...) {},
```

Comment:
Noting that in my own usages of `@fun`, I think I'd need to have some conditional logic for "if the slot exists, use it, if not, use the thing itself." Wondering if we should include a `@fun` [accessor](https://github.com/RConsortium/S7/blob/db5964265a88d114fb6de97361060979ad63b3d6/vignettes/spec/design.Rmd#L159) with a deprecation message?

Reply:
Unfortunately I can't add a deprecation because of a bug in waldo (https://github.com/r-lib/waldo/issues/219) 😞